usagov/vote-gov-drupal
# VOTE-3126: Enable saml auth encryption

## Jira ticket

[VOTE-3126](https://cm-jira.usa.gov/browse/VOTE-3126)

## Description

Enable encryption for saml auth. Update production to use new GSAauth with Encryption enabled.

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. Once deployed to environment, confirm that you are able to login using new GSAauth.

## Checklist for the Developer




## Checklist for the Peer Reviewers
